app/services/request_reference.rb

```diff
@@ -31,6 +29,6 @@ def auto_approve_reference(reference)
   end
 
   def email_address_is_a_bot?(reference)
-    REFEREE_BOT_EMAIL_ADDRESSES.include?(reference.email_address)
+    /^refbot(\d+)@example.com/ =~ reference.email_address
```

Comment:
Consider moving this to a constant, just so it's clearer what the regexp is trying to match